provision.yaml

```diff
@@ -63,4 +63,4 @@ release_checks_7:
   - debian-9-x86_64
   - debian-10-x86_64
   - ubuntu-1804-x86_64
-  - ubuntu-2004-x86_64
+  - ubuntu-2004-x86_64
```

Comment:
Please revert and check your editor config to avoid this in the future.

Reply:
fixed.